stdlib/public/Platform/ucrt.modulemap

```diff
@@ -26,31 +26,48 @@ module _complex [system] [no_undeclared_includes] {
 }
 
 module _stddef [system] [no_undeclared_includes] {
-  header "stddef.h"
-  export *
+  use corecrt
+  textual header "stddef.h"
 }
 
 module _stdlib [system] [no_undeclared_includes] {
   use corecrt
+  use vcruntime
   header "stdlib.h"
   export *
 }
 
+module _ctype [system] [no_undeclared_includes] {
+  use corecrt
+  textual header "ctype.h"
+}
+
+module _errno [system] [no_undeclared_includes] {
+  use corecrt
+  textual header "errno.h"
+}
+
+module _stdnoreturn [system] [no_undeclared_includes] {
```

Comment:
Where is this module used? Especially given the `ucrt.stdnoreturn` module, this seems unused.